src/backend/aspen/covidhub_import/implementation.py

```diff
@@ -103,8 +120,23 @@ def import_project(
         group: Group = session.query(Group).filter(Group.id == aspen_group_id).one()
 
         logger.info("Retrieving from COVIDHUB DB...")
-        consensus_genomes: Collection[ConsensusGenome] = (
-            covidhub_session.query(ConsensusGenome)
+        group_czbids: Collection[CZBID] = (
+            covidhub_session.query(CZBID)
+            .join(Project)
+            .filter(Project.rr_project_id == rr_project_id)
```

Comment:
So the rr_project id is different for internal/external samples though, ex. `RR0092i` vs `RR0092e` am I reading this wrong or does it assume they are the same?

Reply:
We'd run the import twice -- once for `RR0092e` and once for `RR0092i`

Reply:
ahhhhh then all good!